src/reducers/books.js

```diff
@@ -0,0 +1,19 @@
+import {CREATE_BOOK, REMOVE_BOOK} from "../actions";
+import {Book} from "../logic/book";
+import {BOOK_CATEGORIES} from "../logic/book";
+
+const initialState = [
```

Comment:
Great job! This makes your store much simpler :)

Reply:
Thanks